EuniceGatehi/Yummy_Recipes
# #[152454553 ][This PR combines features of Challenge two.]

**What does this PR do?**
- Creates classes, objects and methods recipes and their categories to achieve CRUD operations
- Includes comments and docstrings to code for good coding practice
- Implements the technical feedback received from my LFA's(both for week 1 and 2)
- Obtains badges for build passing[![Build Status](https://travis-ci.org/EuniceGatehi/Yummy_Recipes.svg?branch=developchallengetwo)](https://travis-ci.org/EuniceGatehi/Yummy_Recipes) and coveralls [![Coverage Status](https://coveralls.io/repos/github/EuniceGatehi/Yummy_Recipes/badge.svg?branch=developchallengetwo)](https://coveralls.io/github/EuniceGatehi/Yummy_Recipes?branch=developchallengetwo) through Travis integration
- Deploys app to Heroku

**Description of Task to be completed?**
As indicated in the challenge 2 expectations:
- Setup unit testing libraries and ensure minimal tests written, to practice Test-Driven Development.
- Integrate TravisCI for Continuous Integration in your repository (with ReadMe badge).
- Integrate test coverage reporting (e.g. Coveralls) with a badge in the ReadMe.
- Obtain CI badges (e.g. from Code Climate and Coveralls) and add to ReadMe.
- Integrate the UI templates developed in challenge 1 into your Flask application and make them accessible when you run your app. 
- Develop all required features including their user interfaces. Use OOP and no database (data should be lost when the server is restarted).
- Create a pull request to your development branch and request two of your friends to review it.
- Deploy your Flask application to Heroku.

**How should this be manually tested?**
For the CRUD operations, run https://yummyflask.herokuapp.com/ to see if all features work correctly
For the readme badges, on my developchallengetwo branch, navigate to the README.md file and view the badges
**What are the relevant pivotal tracker stories?**
[Finishes #152454553](https://www.pivotaltracker.com/story/show/152454553)
[Finishes #152062326](https://www.pivotaltracker.com/story/show/152062326)
[Finishes #152181931](https://www.pivotaltracker.com/story/show/152181931)
[Finishes #152181934](https://www.pivotaltracker.com/story/show/152181934)

**Screenshots (if appropriate)**
Images for the Travis and Coveralls reports
![image](https://user-images.githubusercontent.com/18480397/32251826-ca22e79e-bea3-11e7-9bec-45dbb2cad123.png)
![image](https://user-images.githubusercontent.com/18480397/32251919-2bd394f2-bea4-11e7-9736-80c0f84d334d.png)
Screenshot for my Heroku app as it appears when you click this link
https://yummyflask.herokuapp.com/
![image](https://user-images.githubusercontent.com/18480397/32252319-e9975fa4-bea5-11e7-9d40-e0f5a9baac4f.png)
